src/Gui/Stylesheets/CMakeLists.txt

```diff
@@ -5,12 +5,14 @@ SET(Stylesheets_Files
     "Dark-green.qss"
     "Dark-orange.qss"
     "Darker-blue.qss"
+    "Dark-contrast.qss"
     "Darker-green.qss"
     "Darker-orange.qss"
     "Light-blue.qss"
     "Light-green.qss"
     "Light-orange.qss"
     "Dark-contrast.qss"
+    "ProDark.qss"
```

Comment:
I think there was some confusion because Dark-contrast.qss was added twice in the list.
The order is also not alphabetical (which doesn't have any impact anyway)